### ISSUE TITLE
[Feature] OTA updates

### ISSUE THREAD

**Issue by cobaltgit** [cizia64/CrossMix-OS#61]
Just had a thought, OTA updates to Crossmix would be great for those who want to update their console without plugging their SD card into a computer, for example the console checks for a WiFi connection then downloads the latest release, unzips and overwrites then reboots.

**Comment by ogregorio** [comment]
+ 1

**Comment by slrgt** [comment]
this would be great

**Comment by cizia64** [comment]
Yes good idea... but the first step is to have an update script.
I have modified so much stuffs on the v1.2.0 that we can't update just by copying the files as for the v1.1.0 update.
So the idea is to make an update script to keep only roms,bios and saves over a fresh v1.2.0. 
Then if we have a good update script we can plan to make OTA.
